.github/workflows/build_rpms.yaml

```diff
@@ -0,0 +1,52 @@
+name: Build RPMs
+
+on:
+  schedule:
+  - cron: '0 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  build_rpm_build_container:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up registry credentials
+      if: ${{ github.event_name != 'pull_request' }}
```

Comment:
I think this isn't quite right because this will also run on branches pushed to local forks.  We probably want something that limits this only to the official repo? or perhaps write it in such a way that the code runs anywhere, but only executes if the values are set... `if [ -n "${{ secrets.REGISTRY_USERNAME }}"]; then`...